diff --git a/core/templates/update.admin.php b/core/templates/update.admin.php
index 87f9396..368a533 100644
--- a/core/templates/update.admin.php
+++ b/core/templates/update.admin.php
@@ -37,7 +37,7 @@
 		<input class="updateButton" type="button" value="<?php p($l->t('Start update')) ?>">
 		<div class="infogroup">
 			<?php p($l->t('To avoid timeouts with larger installations, you can instead run the following command from your installation directory:')) ?>
-			<pre>./occ upgrade</pre>
+			<pre>sudo -u apache php /usr/share/nextcloud/occ upgrade</pre>
 		</div>
 	</div>
 
diff --git a/core/templates/update.use-cli.php b/core/templates/update.use-cli.php
index 52d40cd..945d4d9 100644
--- a/core/templates/update.use-cli.php
+++ b/core/templates/update.use-cli.php
@@ -7,8 +7,10 @@
 			} else {
 				p($l->t('Please use the command line updater because automatic updating is disabled in the config.php.'));
 			} ?><br><br>
-			<?php
-			print_unescaped($l->t('For help, see the  <a target="_blank" rel="noreferrer" href="%s">documentation</a>.', [link_to_docs('admin-cli-upgrade')])); ?><br><br>
+      <code>
+sudo -u apache php /usr/share/nextcloud/occ upgrade
+</code>
+			<br><br>
 		</div>
 	</div>
 </div>
diff --git a/updater/index.php b/updater/index.php
index 017ca70..085c250 100644
--- a/updater/index.php
+++ b/updater/index.php
@@ -1830,7 +1830,7 @@ if(strpos($updaterUrl, 'index.php') === false) {
 			var el = document.getElementById('step-maintenance-mode')
 				.getElementsByClassName('output')[0];
 			if (keepActive) {
-				el.innerHTML = 'Maintenance mode will kept active.<br>Now trigger the migration via command line: <code>./occ upgrade</code><br>';
+				el.innerHTML = 'Maintenance mode will kept active.<br>Now trigger the migration via command line: <code>sudo -u apache php /usr/share/nextcloud/occ upgrade</code><br>';
 				successStep('step-maintenance-mode');
 				currentStep('step-done');
 				performStep(11, performStepCallbacks[11]);
