diff --git a/console.php b/console.php
index b2724db..149c265 100644
--- a/console.php
+++ b/console.php
@@ -31,22 +31,6 @@ use Symfony\Component\Console\Output\ConsoleOutput;
 
 define('OC_CONSOLE', 1);
 
-// Show warning if a PHP version below 5.4.0 is used, this has to happen here
-// because base.php will already use 5.4 syntax.
-if (version_compare(PHP_VERSION, '5.4.0') === -1) {
-	echo 'This version of Nextcloud requires at least PHP 5.4.0'.PHP_EOL;
-	echo 'You are currently running ' . PHP_VERSION . '. Please update your PHP version.'.PHP_EOL;
-	return;
-}
-
-// Show warning if PHP 7.1 is used as Nextcloud is not compatible with PHP 7.1 for now
-// @see https://github.com/nextcloud/docker-ci/issues/10
-if (version_compare(PHP_VERSION, '7.1.0') !== -1) {
-	echo 'This version of Nextcloud is not compatible with PHP 7.1.<br/>';
-	echo 'You are currently running ' . PHP_VERSION . '.';
-	return;
-}
-
 function exceptionHandler($exception) {
 	echo "An unhandled exception has been thrown:" . PHP_EOL;
 	echo $exception;
diff --git a/cron.php b/cron.php
index 4473dba..7a4e1a6 100644
--- a/cron.php
+++ b/cron.php
@@ -30,20 +30,6 @@
  *
  */
 
-// Show warning if a PHP version below 5.4.0 is used
-if (version_compare(PHP_VERSION, '5.4.0') === -1) {
-	echo 'This version of Nextcloud requires at least PHP 5.4.0<br/>';
-	echo 'You are currently running ' . PHP_VERSION . '. Please update your PHP version.';
-	return;
-}
-
-// Show warning if PHP 7.1 is used as Nextcloud is not compatible with PHP 7.1 for now
-// @see https://github.com/nextcloud/docker-ci/issues/10
-if (version_compare(PHP_VERSION, '7.1.0') !== -1) {
-	echo 'This version of Nextcloud is not compatible with PHP 7.1.<br/>';
-	echo 'You are currently running ' . PHP_VERSION . '.';
-	return;
-}
 
 try {
 
diff --git a/index.php b/index.php
index e72d38c..29920d7 100644
--- a/index.php
+++ b/index.php
@@ -25,22 +25,6 @@
  *
  */
 
-// Show warning if a PHP version below 5.4.0 is used, this has to happen here
-// because base.php will already use 5.4 syntax.
-if (version_compare(PHP_VERSION, '5.4.0') === -1) {
-	echo 'This version of Nextcloud requires at least PHP 5.4.0<br/>';
-	echo 'You are currently running ' . PHP_VERSION . '. Please update your PHP version.';
-	return;
-}
-
-// Show warning if PHP 7.1 is used as Nextcloud is not compatible with PHP 7.1 for now
-// @see https://github.com/nextcloud/docker-ci/issues/10
-if (version_compare(PHP_VERSION, '7.1.0') !== -1) {
-	echo 'This version of Nextcloud is not compatible with PHP 7.1.<br/>';
-	echo 'You are currently running ' . PHP_VERSION . '.';
-	return;
-}
-
 try {
 
 	require_once __DIR__ . '/lib/base.php';
diff --git a/settings/Controller/CheckSetupController.php b/settings/Controller/CheckSetupController.php
index 86b9b76..ac42177 100644
--- a/settings/Controller/CheckSetupController.php
+++ b/settings/Controller/CheckSetupController.php
@@ -212,9 +212,7 @@ class CheckSetupController extends Controller {
 	 * @return bool
 	 */
 	protected function isPhpOutdated() {
-		if (version_compare(PHP_VERSION, '5.5.0') === -1) {
-			return true;
-		}
+    // PHP supported in distribution
 		return false;
 	}
 
